README.md

```diff
@@ -1,5 +1,10 @@
 # xrdocument
-OpenXRLab Document Address : http://doc.openxrlab.org.cn/openxrlab_docment/ 
 
+**Welcome to OpenXRLab!**
 
+## Introdunction 
+xrdocoment will offer the summary intro of the openxrlab group and the index of every repo.
+in addition in the doc we have made a while AR demo by xrslam xrlocalization and xrsfm. you can refer the doc to build it and check the result of demo.
```

Comment:
In addition, we provide a AR demo by connecting [XRSLAM](), [XRSfM]() and [XRLocalization](). Refer to the [doc]() to build it and play around the demo!

Reply:
have modified again, according to your suggestion